src/utils/preprocessing/preprocessing.js

```diff
@@ -13,10 +15,17 @@ function tokenize(sentence) {
   return sentence.split(" ");
 }
 
+function isExpression(word) {
```

Comment:
it's funny that the variable is named `word` ...  maybe we should name it `text`? or something that allows for either word or expression?